src/bento/programData.js

```diff
@@ -1,12 +1,13 @@
 import gql from 'graphql-tag';
+import copProgramImage from '../assets/programs/Program_Image_COP.png';
 
 const pageData = {
   headerTitle: 'Programs',
   headerIcon: 'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/assets/icons/Icon-Programs.svg',
   externalIcon: 'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/assets/icons/Program-ExternalLink.svg',
   COP: {
     prgramName: 'COP',
-    primaryImage: 'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/assets/programCards/cop/copPrimary.png',
+    primaryImage: copProgramImage,
```

Comment:
@rana22 Can you push image to master branch under "src/assets/programCards/cop/" and put raw.github link here.

Reply:
I have created PR168.